docs/cloud-tutorials/application-management.md

```diff
@@ -40,7 +47,9 @@ If your account belongs to an [organization](./account-management.md#organizatio
 If you edit your application or schema, run `npx dbos-cloud app deploy` again to apply the latest migrations and upgrade to the latest version.
 
 :::info
-You don't have to worry about changing database server connection parameters like `hostname` or `password` in [`dbos-config.yaml`](../api-reference/configuration.md) to deploy an application to the cloud&#8212;DBOS automatically applies the connection information of your cloud database instance.
+* You don't have to worry about changing database server connection parameters like `hostname` or `password` in [`dbos-config.yaml`](../api-reference/configuration.md) to deploy an application to the cloud&#8212;DBOS automatically applies the connection information of your cloud database instance.
+* You cannot change the database name of a deployed application. You can delete and re-deploy the application.
+* Databases cannot be shared across applications.
```

Comment:
This **might** need clarification/differentiation between database instance and app db name, similar to what you did above.